app/decorators/generic_object_decorator.rb

```diff
@@ -4,6 +4,6 @@ def self.fonticon
   end
 
   def fileicon
-    try(:picture) ? "/pictures/#{picture.basename}" : "100/generic_object.png"
+    try(:picture) ? "/pictures/#{picture.basename}" : nil
```

Comment:
@skateman What about ff-generic-object ?

Reply:
@epwinchell that's being used, see above. But if the `picture` is defined, it overrides the `fonticon`.